### PR TITLE
Add bundle errata releases to 2.4 release notes

### PR DESCRIPTION
**Pull request by michellemacrh**
Add the Ansible bundle errata releases to the 2.4 Release Notes

https://issues.redhat.com/browse/AAP-16768

### CODE REVIEW

File: downstream/titles/release-notes/topics/errata-24-aug-28.adoc [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.Bug fixes
+
+* Changing credential types by using the drop-down list in the *Launch prompt* window no longer causes the screen to disappear.
+
+* Upgraded python dependencies which include upgrades from Django 3.2 to 4.2.3, psycopg2 to psycopg3, and additional libraries as needed. Also added a new setting in the UI exposing the `CSRF_TRUSTED_ORIGIN` settings.
+
+* Fixed slow database UPDATE statements on the job events table which could cause a task manager timeout.
+
+* Adding new labels to a job through prompting now works as expected.
```
[the remark this answers is not in the document]
Comment (michellemacrh) [reply downstream/titles/release-notes/topics/errata-24-aug-28.adoc:27]: I agree this is a bit confusing to read. I can try to dig out the associated Jira ticket for this item to see if I can get more clarity and reword it 👍

File: downstream/titles/release-notes/topics/errata-24-aug-28.adoc [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+* Activation statuses are now reported correctly, whether you are disabling or enabling them.
+
+* When the `run_job_template` action fails, ansible-rulebook prints an error log in the activation output and creates an entry in rule audit so the user is alerted that the rule has failed.
+
+* When a user tries to bulk delete rulebook activations from the list, the request now completes successfully and consistently.
+
+* The *Rulebook Activation* link now functions correctly in the *Rule Audit Detail* UI.
+
+* Fixed a bug where ansible-rulebook prevented the execution if the connection with the controller was not successful when controller was not required by the rulebook.
```
[the remark this answers is not in the document]
Comment (michellemacrh) [reply downstream/titles/release-notes/topics/errata-24-aug-28.adoc:95]: I'll take a look at the Jira to get more clarity and reword 👍

File: downstream/titles/release-notes/topics/errata-24-aug-28.adoc [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+.Bug fixes
+
+* Changing credential types by using the drop-down list in the *Launch prompt* window no longer causes the screen to disappear.
+
+* Upgraded python dependencies which include upgrades from Django 3.2 to 4.2.3, psycopg2 to psycopg3, and additional libraries as needed. Also added a new setting in the UI exposing the `CSRF_TRUSTED_ORIGIN` settings.
+
+* Fixed slow database UPDATE statements on the job events table which could cause a task manager timeout.
+
+* Fixed an issue where adding a new label to a job through the *Prompt On Launch* option would not add the label to the job details.
```
Comment (emurtoug) [inline downstream/titles/release-notes/topics/errata-24-aug-28.adoc:27]: This is an awesome resolution to the confusion of this line before! Super job.

File: downstream/titles/release-notes/topics/errata-24-aug-10.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+* Added the ability to add execution and hop nodes to VM-based controller installations from the UI.
+
+* Added the `awx-manage` command for creating future events table partitions.
+
+* Re-enabled Pendo support by providing the correct Pendo API key.
+
+* Added the ability to filter teams by using partial names in the dialog for granting teams access to a resource.
+
+* Fixed a bug where a weekly rrule string without a BYDAY value would result in the UI throwing a TypeError.
```
Comment (emurtoug) [inline downstream/titles/release-notes/topics/errata-24-aug-10.adoc:43]: Just a line edit here, "rrule" should be "rule".

File: downstream/titles/release-notes/topics/errata-24-aug-28.adoc [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+* Activation statuses are now reported correctly, whether you are disabling or enabling them.
+
+* When the `run_job_template` action fails, ansible-rulebook prints an error log in the activation output and creates an entry in rule audit so the user is alerted that the rule has failed.
+
+* When a user tries to bulk delete rulebook activations from the list, the request now completes successfully and consistently.
+
+* The *Rulebook Activation* link now functions correctly in the *Rule Audit Detail* UI.
+
+* The ansible-rulebook now only connects to the controller if the rulebook being processed has a `run_job_template` action.
```
Comment (emurtoug) [inline downstream/titles/release-notes/topics/errata-24-aug-28.adoc:95]: Great resolution here, too!

File: downstream/titles/release-notes/topics/errata-24-aug-28.adoc (hunk 3 above)
Comment (michellemacrh) [reply downstream/titles/release-notes/topics/errata-24-aug-28.adoc:27]: Thanks so much for saying so :D

File: downstream/titles/release-notes/topics/errata-24-aug-10.adoc (hunk 4 above)
Comment (michellemacrh) [reply downstream/titles/release-notes/topics/errata-24-aug-10.adoc:43]: I may be wrong but my understanding from reading the Jira ticket is that rrule is a specific attribute (I'll send you on a screenshot of the ticket)

**Pull request merged**